shared/oval/package_audit_installed.xml

```diff
@@ -5,11 +5,14 @@
     <metadata>
       <title>Package audit Installed</title>
       <affected family="unix">
-        <platform>multi_platform_rhel</platform>
+        <platform>CentOS 4</platform>
+        <platform>CentOS 5</platform>
```

Comment:
On 2/21/16 4:17 PM, Ján Lieskovský wrote:
>
> In shared/oval/package_audit_installed.xml 
> <https://github.com/OpenSCAP/scap-security-guide/pull/1048#discussion_r53573538>:
>
> > @@ -5,11 +5,14 @@
> >      <metadata>
> >        <title>Package audit Installed</title>
> >        <affected family="unix">
> > -        <platform>multi_platform_rhel</platform>
> > +        <platform>CentOS 4</platform>
> > +        <platform>CentOS 5</platform>
>
> @shawndwells <https://github.com/shawndwells>
>
>     If RHEL6, should CentOS6 be included too?
>
> No. This change isn't adding support for |CentOS 6|. It's just 
> combining the platform tags from 
> |RHEL/5/input/oval/package_audit_installed.xml| and |Red Hat 
> Enterprise Linux 6| part from 
> |shared/oval/package_audit_installed.xml|. |Red Hat Enterprise Linux 
> 7| platform needs to be split into separate oval under 
> |shared/oval/oval_5.11| directory because |package_audit_installed| is 
> not a standalone OVAL. It's used only as a prerequisite OVAL for 
> |service_auditd_enabled|, which should be included only in moment 
> oscap on system supports OVAL-5.11 (otherwise we will have one unused 
> OVAL when using 5.10 and |make validate| will complain).
>

Asked a different way: On lines 10-12 you have added CPE tags for RHEL4, 
RHEL5, and RHEL6. On lines 8 and 9, you have added CentOS 4 and CentOS 5.

Is there a reason RHEL6 was added, but not CentOS 6? Should we take this 
PR as an opportunity to add CentOS6 CPE tags?